test/tests/imageserver/README.md

```diff
@@ -0,0 +1,54 @@
+# External File Server(On-static) Test:
+
+This test validate the function of RackHD image service. There are 3 parts of test:
+###1. Test OS image service
+This test OS image operation related APIs of image-service.
+###2. Test microkernel service
+This test microkernel management related APIs of image-service
+###3. Test bootstraping and rediscovery
+This test RackHD+ image service working together to do OS bootstrapping and node rediscovery.
+
+##Config file
+You need put an config file in the /config directory containing the test file url. The test date should write in json format and save as fileserver_config.jon. The following is an example.
+The first part contains external OS image repos, the second part contains microkernel image repos, the third part is the credential of fileserver used for testing local iso mount.
+When test the fileserver, you need set a environment variable $IMAGESERVER first, then you can run the FIT test. It is need to specify the fileserver's credential by "usr" and "pwd" and "supwd" is the test server's admin user's password
+###filetocompare
+This is the parameter to specify the max file to go through in each repo. Some iso repo may have thousands of files. It unnecessary to go though all of them. 
+###rackhd_control_ip
+This parameter is the south bound nic ip of RackHD. It is 172.31.128.1 be default.
+
+##FIT parameter
+-imageserver is used to specify the image service server IP. Use as "-imageserver=xx.xx.xx.xx"
+
+##Example 
+python run_tests.py -test ./tests/fileserver/test_os_static_file_server.py  -rackhd_host XX.XX.XX.XX  -imageserver=xx.xx.xx.xx
+
+Sample global_config.json file:

```

Comment:
global_config.json file doesn't exist.  Do you want to instead create a test specific install_config.json file? that config files contains links for OS install images.